<div class="main-content">
    <div class="container-fluid">
        <div class="row">
            <div class="col-md-12">
                <div class="card">
                    <br>
                    <div class="card-body">
                        <mat-tab-group>
                            <mat-tab label="Mouse Variant Allele Registry">
                                <h3>MOUSE VARIANT ALLELE REGISTRY</h3>

                                <p> Model organisms are essential to understanding the biological and disease
                                    consequences of human genome variation. Bioinformatics resources that support
                                    meaningful comparisons of mouse and human genotype-to-phenotype data and knowledge
                                    are needed to support the translation from bench to bedside and back again <b>[1]</b>.
                                    There is no genome variation resource for mouse comparable to resources available
                                    for human genome variation data such as EXAC <b>[2]</b>, ClinVar <b>[3]</b>, or ClinGen 
                                    <b>[4]</b>. NCBI resources such as dbSNP and ClinVar no longer accept data from model organisms.
                                    While the European Bioinformatics Institute (EBI) will be accepting SNP data for
                                    mouse, the resource will not include all relevant mouse variation data and will not
                                    include all of the mouse-specific information (functional and phenotypic attributes)
                                    that are central to data interpretation and analysis.
                                </p>

                                <p>
                                    Although the Mouse Genome Informatics database (MGI) <b>[5]</b> serves as a comprehensive
                                    mouse allele registry, the data are not structured in a format consistent with the
                                    <a href="http://varnomen.hgvs.org/">Human Genome Variation Society (HGVS)</a>
                                    format. Furthermore, not all sources of mouse variation reside in MGI. For example, the
                                    Mutant Mouse Resource (MMR) created a stand-alone database for variants discovered
                                    via large-scale exome sequencing of mutant mice with Mendelian disease phenotypes
                                    but these data cannot currently be integrated with variation data in MGI because of
                                    data format differences. Plans to establish a collaborative effort on function of
                                    variation in long non-coding RNAs between MGI and the Robinson laboratory cannot
                                    proceed efficiently until the representation of sequence level variation in MGI is
                                    possible. GeneWeaver <b>[6]</b> currently incorporates gene-level annotations from MGI but
                                    is currently incorporating sequence variation incrementally because of the lack of a
                                    central repository of these data for mouse. MGI can only integrate with the JAX CKB <b>[7]</b>
                                    resource for cancer model information at the gene level because of the lack of
                                    sequence variant representation.
                                </p>
                                <p>
                                    The lack of a genome sequence variation resource for mouse is a significant barrier
                                    to comparing variation and its biological consequences between mouse and human and
                                    limits the impact of many research and resource development programs within JAX.
                                    This DIF project seeks to address this resource gap by bringing together
                                    investigators that have active projects in the area of genome variation in either
                                    mouse or human or both. Many of the investigators on this project have developed
                                    independent resources to curate or manage genome variation. This project aims to
                                    unify these efforts and build a common data resource.

                                </p>
                                <ol>
                                    <li>Manolio, T.A., et al., <i>Bedside Back to Bench: Building Bridges between Basic and
                                        Clinical Genomic Research.</i> Cell, 2017. <b>169</b>(1): p. 6-12.</li>
                                    <li>Karczewski, K.J., et al., <i>The ExAC browser: displaying reference data information 
                                        from over 60 000 exomes.</i> Nucleic Acids Res, 2017. <b>45</b>(D1): p.D840-D845.</li>
                                    <li>Landrum, M.J., et al., <i>ClinVar: public archive of  relationships among sequence 
                                       variation and human phenotype.</i> Nucleic Acids Res, 2014. <b>42</b>(Database issue): p. D980-5.</li>
                                    <li>Rehm, H.L., et al., <i>ClinGen--the Clinical Genome Resource.</i> N Engl J Med, 2015. <b>372</b>
                                        (23): p. 2235-42.</li>
                                    <li>Bult, C.J., et al., <i>Mouse genome database 2016.</i> Nucleic Acids Res, 2016. <b>44</b>
                                        (D1): p. D840-7.</li>
                                    <li>Bubier, J.A., et al., <i>GeneWeaver: finding consilience in heterogeneous cross-species                  functional 
                                       genomics data.</i> Mamm Genome, 2015. <b>26</b>(9-10): p. 556-66.</li>
                                    <li>Patterson, S.E., et al., <i>The clinical trial landscape in oncology and connectivity of
                                        somatic mutational profiles to targeted therapies.</i> Hum Genomics, 2016. <b>10</b>: p. 4.</li>
                                </ol>
                            </mat-tab>
                            <mat-tab label="Team">
                                <div></div>
                            </mat-tab>


                            <mat-tab label="Publications">
                                <div></div>
                            </mat-tab>

                            <mat-tab label="Contact Us">
                                <div></div>
                            </mat-tab>

                        </mat-tab-group>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>