<div mat-dialog-content>
    <div class="row">
        <div class="col-lg-6 col-md-6">
            <h3 *ngIf="helpType ==='SNPandIndelGeneral'">SNPs/INDELs</h3>
        </div>
        <div class="col-lg-6 col-md-6">
                <span class="pull-right align-content-end">
                    <a mat-button class="nav-link active" (click)="closeDialog()">
                      <i class="material-icons">close</i> Close
                      <div class="ripple-container"></div>
                    </a>
                </span>
        </div>
    </div>


    <div *ngIf="helpType ==='SNPandIndelGeneral'" class="row">
        <div class="col-lg-12 col-md-12">

            <p><br>Variant calls from reference (GRCm38) based alignment of Illumina short read data (exome and whole
                genome), and generated from a standard GATK pipeline with modification for laboratory mouse genomes
                (Fairfield et al., 2015).</p>

            <p><strong>Position</strong> – the physical position of the variant in the mouse genome reference build
                GRCm38</p>

            <p><strong>Type</strong> – the type of variant, SNP: single nucleotide polymorphism, DEL: deletion, INS:
                insertion</p>

            <p><strong>Impact</strong> – predicted impact annotation, e.g. from SNPEff, Putative_impact: A simple
                estimation of putative impact / deleteriousness : (HIGH, MODERATE, LOW, MODIFIER), reference <a
                        href="http://snpeff.sourceforge.net/SnpEff_manual.html" rel="noopener noreferrer"
                        target="_blank">http://snpeff.sourceforge.net/SnpEff_manual.html</a>
                for details</p>

            <p><strong>Annotation</strong> – primary annotation described by the Sequence Ontologies, which is a set of terms and relationships used
                to describe the features and attributes of biological sequence. SO includes different kinds of features which can be located on the 
                sequence. Biological features are those which are defined by their disposition to be involved in a biological process. Examples are 
                <b>binding_site</b> and <b>exon</b>. More details can be found here:                <a
                        href="http://www.sequenceontology.org/" rel="noopener noreferrer"
                        target="_blank">http://www.sequenceontology.org/</a>
            </p>

        </div>
    </div>

    <div *ngIf="helpType ==='SVGeneral'" class="row">
        <div class="col-lg-12 col-md-12">

            <p><br>Structural variant calls from reference (GRCm38) based on whole genome sequence data, and generated
                from
                a consensus structural variant calling pipeline based on 4 callers (Breakdancer, Delly, Lumpy, and
                Manta) for laboratory mouse genomes.</p>

            <p><strong>Type</strong> – Structural variant type classed into the following:</p>
            <ul>
                <li>INS – Insertion</li>
                <li>INV – Inversion</li>
                <li>DEL – Deletion</li>
                <li>DUP – Duplication</li>
                <li>TRA – Translocation</li>
            </ul>


            <p><strong>Start Position</strong> – Starting position of the SV call. Note for translocation events this
                position represents a single breakpoint, rather than the start of a range.</p>

            <p><strong>End Position</strong> – Ending position of the SV call. Note for translocation events this
                position represents a single breakpoint, rather than the end of a range.</p>

            <p><strong>Length</strong> – Size of the SV event. Note: For Insertion calls accurate size prediction is not
                possible with available tools. Rather start and end position show region where an insertion is predicted
                to be present. For translocations, start and end position represent predicted locations of where a
                translocation event has occurred, and size prediction is not possible.</p>

            <p><strong>Prop. Occurrence</strong> - The frequency of the structural variant within the MMRdb variant
                database.

            <p><strong>Supporting Callers</strong> – The number of structural variant callers supporting each call.
                Total possible callers are four.</p>

            <p><strong>Mutation Candidate</strong> – A variant is a likely pathogenic (y) if its occurrence is low (0-1%
                frequency where variants found in a single sample are ~0%), its genotype is supported by the phenotype
                of the sample and the inheritance of the phenotype (e.g. for autosomal recessive phenotypes, an affected
                individual must be homozygous), and if linkage information is available, the variant must be linked</p>

            <p><strong>Sample ID</strong> – The ID assigned to each sample when it is added to MMRdb. Use this I.D. to
                search for additional sample information under ‘Samples’</p>


        </div>
    </div>
</div>