<div mat-dialog-content>

    <div class="row">
        <div class="col-lg-12 col-md-12">
    <span class="pull-right">
        <a mat-button class="nav-link active" (click)="closeDialog()">
          <i class="material-icons">close</i> Close
          <div class="ripple-container"></div>
        </a>
    </span>
        </div>
    </div>
    <div *ngIf="helpType ==='SNPandIndelGeneral'" class="row">
        <div class="col-lg-12 col-md-12">
            <h5><strong>SNPs/INELs</strong></h5>
            <p> Variant calls from reference (GRCm38) based alignment of Illumina short read data (exome and whole
                genome), and generated from a standard GATK pipeline with modification for laboratory mouse genomes
                (Fairfield et al., 2015).</p>

            <p><strong>Position</strong> – the physical position of the variant in the mouse genome reference build
                GRCm38</p>

            <p><strong>Type</strong> – the type of variant, SNP: single nucleotide polymorphism, DEL: deletion, INS:
                insertion</p>

            <p><strong>Impact</strong> – predicted impact annotation, e.g. from SNPEff, Putative_impact: A simple
                estimation of putative impact / deleteriousness : (HIGH, MODERATE, LOW, MODIFIER), reference <a
                        href="http://snpeff.sourceforge.net/SnpEff_manual.html" rel="noopener noreferrer" target="_blank">http://snpeff.sourceforge.net/SnpEff_manual.html</a>
                for details</p>

            <p><strong>Functional Class</strong> – predicted variant function according to position and base change,
                e.g. from SNPEff, Functional class (NONE, SILENT, MISSENSE, NONSENSE), reference <a
                        href="http://snpeff.sourceforge.net/SnpEff_manual.html" rel="noopener noreferrer" target="_blank">http://snpeff.sourceforge.net/SnpEff_manual.html</a>
                for details</p>

            <p><strong>Occurrence %</strong> - the frequency of the variant within the MMRdb variant database by variant
                position</p>

            <p><strong>Mutation Candidate</strong> – A variant is a mutation candidate if its occurrence is low (0-1%
                frequency where variants found in a single sample are ~0%), its genotype is supported by the phenotype
                of the sample and the inheritance of the phenotype (e.g. for autosomal recessive phenotypes, an affected
                individual must be homozygous), and if linkage information is available, the variant must be linked)</p>

            <p><strong>Sample ID</strong> – The ID assigned to each sample when it is added to MMRdb. Use this I.D. to
                search for additional sample information under ‘Samples’</p>

        </div>
    </div>
    <div *ngIf="helpType ==='sampleGeneral'" class="row">
        <div class="col-lg-12 col-md-12">
            <h5><strong>Samples</strong></h5>

            <p><strong>Sample ID</strong> – The ID assigned to each sample when it is added to MMRdb. Use this I.D. to
                search for additional sample information under ‘Samples’</p>

            <p><strong>Sample Name</strong> – The name of the sample provided by the owner of the sample / data</p>

            <p><strong>JAX Registry ID</strong> – This is the JR or JAX strain ID which can be used to search the
                Jackson Laboratory Strain database or the Mouse Genome Database for additional strain information. Link
                out to the JAX Strain datasheet and ordering information</p>

            <p><strong>LIMS sample ID</strong> – This is the sample ID that can be used to track sample preparation and
                sequencing information if the data were generated by Genome Technologies at The Jackson Laboratory</p>

            <p><strong>Chromosome Linkage</strong> – If mapping data are available for the phenotype, this is the
                chromosome containing the SNPs with the most significant LOD scores</p>

            <p><strong>Genotype</strong> – this is inferred (if possible) from inheritance and phenotypic status of the
                sample</p>

            <p><strong>Study</strong> – This is the identifier for the study from which the data were generated. WES:
                whole exome sequencing (paired end is default), WGS: whole genome sequencing (paired end is default),
                MP: mate pair or long jump library, SE: single end</p>

            <p><strong>Platform</strong> – relevant to whole exome sequencing, this is the version of the exome capture
                reagents</p>

            <p><strong>Likely pathogenic</strong> – A variant is a likely pathogenic (y) if its occurrence is low (0-1%
                frequency where variants found in a single sample are ~0%), its genotype is supported by the phenotype
                of the sample and the inheritance of the phenotype (e.g. for autosomal recessive phenotypes, an affected
                individual must be homozygous), and if linkage information is available, the variant must be linked)</p>

            <p><strong>Confirmed mutations</strong> – This is reserved for those mutation candidate / likely pathogenic
                variants that have been confirmed through linkage analysis and / or PCR validation to be the putative
                mutation causing the phenotype that is annotated for a given sample. This field is user provided and is
                typically supported by published data</p>

        </div>
    </div>

    <div *ngIf="helpType ==='SVGeneral'" class="row">
        <div class="col-lg-12 col-md-12">
            <h5><strong>Structural Variants</strong></h5>

        </div>
    </div>
</div>